spec/services/form1010cg/service_spec.rb

```diff
@@ -706,26 +707,42 @@
 
       subject = described_class.new(claim, submission)
 
-      expect(subject.claim).to receive(:to_pdf).and_raise('pdf generation failure')
+      error = StandardError.new('Something went wrong')
```

Comment:
You might want to be more specific and build an error class for this service, check `lib/` for other examples of custom exceptions